eng/pipelines/trigger-ml-sample-pipeline.yml

```diff
@@ -61,20 +61,20 @@ jobs:
         Get-ChildItem "$(mlrepo)" -R
       displayName: Output Staging Directory
 
-    - task: PythonScript@0
-      displayName: 'Generate Samples Repo Changes'
-      env:
-        BLOB_CONNECTION_STRING: $(azure-sdk-docsupport-cs)
-      inputs:
-        scriptPath: 'scripts/devops_tasks/generate_ml_sample_update.py'
-        arguments: >-
-          --ml-repo="$(mlrepo)"
-          --ml-wheel-folder="$(Build.ArtifactStagingDirectory)/azure-ai-ml"
-          --build-id="$(Build.BuildId)"
-
     - pwsh: |
         cat "$(mlrepo)/sdk/python/setup.sh"
-      displayName: Display Updated setup.sh
+      displayName: Display setup.sh pre-update
+
+    - task: AzurePowerShell@5
+      displayName: Generate Samples Repo Changes
+      inputs:
+        azureSubscription: 'Azure SDK Engineering System'
+        azurePowerShellVersion: LatestVersion
+        pwsh: true
+        ScriptType: InlineScript
+        Inline: |
+          python scripts/devops_tasks/generate_ml_sample_update.py --ml-repo "$(mlrepo)" --ml-wheel-folder "$(Build.ArtifactStagingDirectory)/azure-ai-ml" --build-id "$(Build.BuildId)" --storage-account-name "$(StorageAccountName)";
```

Comment:
Where is StorageAccountName set? Can it just be a variable in the yml?

Reply:
It's currently set a secret variable on the build def. It's just the name of the storage account so sure.